ixmp/tests/core/test_timeseries.py

```diff
@@ -174,6 +174,18 @@ def test_add_timeseries(ts, format):
         ts.add_timeseries(DATA[0].drop('unit', axis=1))
 
 
+@pytest.mark.parametrize('format', ['long', 'wide'])
+def test_add_timeseries_with_extra_col(ts, format):
+    _data = DATA[0].copy()
+    _data['climate_model'] = [0, 1]
+    data = _data if format == 'long' else wide(_data)
```

Comment:
not sure if `climate_model` will be in `data` after calling `wide`

Reply:
added a check to verify expected behaviour